Sim fit

gkrintir opened the pull request:
vdmDriverII.py, vdmFitterII.py and vdmUtilities.py modified to handle the sim fit.

Four model available:

i)fits/SimCapSigma_PCCAndVtxNoConst_Fit.py (common parameters (Sigma, sigma_{1}/sigma_{2}, Frac) and no const for PCC bkg.)

ii) fits/SimCapSigma_PCCAndVtx_Fit.py (common parameters (Sigma, sigma_{1}/sigma_{2}, Frac) and  const for PCC bkg.)

iii) **Currently recommended**: fits/SimRatio_PCCAndVtxNoConst_Fit.py (common parameter (sigma_{1}/sigma_{2}) and no const for PCC bkg.)

iv) fits/SimRatio_PCCAndVtx_Fit.py (common parameter (sigma_{1}/sigma_{2}) and const for PCC bkg.)

Under investigation, why adding a const term makes the fit unstable.

Also graph files for vertices added (and for PCC modified) to include the first scan pair ;D

gkrintir committed:
SimFit

gkrintir committed:
SimFit

capalmer85 commented on myfcn.C:
```diff
@@ -0,0 +1,122 @@
+/*
```
what is this function for?

gkrintir replied:
This is the way I define the chi2 functions, i.e. the combine function and its Eval() method, and compile them in the respective Sim*.py macro. 
(I have to change it to a more appropriate name, plus to edit its content, because currently its confusing indeed)

capalmer85 commented:
Please remove all .pyc files.

gkrintir committed:
Fixed the SimFit bug; Removed .pyc files

gkrintir commented:
Ok, the final SimFit model is then

fits/SimCapSigma_PCCAndVtx_Fit.py (common parameters are :
- Sigma, sigma_{1}/sigma_{2}, Frac, while the model takes also care about the 
- const for PCC bkg.

Its associated Config file is the fits/SimCapSigma_PCCAndVtx_Config.json ;D

I removed from the repo the other SimFit models that I tried, e.g w/o the const parameter etc.

Also I renamed the function myfnc.C to globalChi2.C, and added comments inside it about the parameter correspondence.

gkrintir committed:
Update README

gkrintir committed:
Fixed small bugs after testing

gkrintir committed:
Add user permission to proceed with the fit in case the GraphsFile extension is different than the Corr argument

capalmer85 commented:
Superseded by #22
